CMakeLists.txt

```diff
@@ -60,6 +60,7 @@ file(TOUCH "${PROJECTM_DUMMY_SOURCE_FILE}")
 
 if(CMAKE_SYSTEM_NAME STREQUAL Emscripten)
     set(ENABLE_EMSCRIPTEN ON CACHE BOOL "Build for web with emscripten. Will also build the SDL2-based entrypoint." FORCE)
+    option(USE_PTHREADS "Enable pthreads" OFF)
```

Comment:
You might consider adding a status message below if Emscripten and USE_PTHREADS is enabled to make it clear when configuring. No change necessary though.